lib/src/request_processor.cc

```diff
@@ -185,6 +191,28 @@ RequestProcessor::RequestProcessor(std::shared_ptr<PXPConnector> connector_ptr,
     }
 
     logLoadedModules();
+
```

Comment:
We provide a way to prevent purging that consists of setting the TTL value to 0, no matter what suffix is used in the TTL string (NB: TTL format = <value><suffix>).

So, here, the getMinutes() method allows to check if such 'value' was set to 0.
I think that having a getMinutes() method, that gives you complete information in any case,  could be more useful than having a value extractor, which would provide only a part of the TTL data.